content/enterprise/before-installing/index.mdx

```diff
@@ -217,7 +217,7 @@ Refer to the sections below for instructions on how to verify your Docker Engine
      https://raw.githubusercontent.com/moby/moby/master/profiles/seccomp/default.json
    ```
 
-1. Edit the `/etc/docker/seccomp.json` file and modify the line `"defaultAction": "SCMP_ACT_ERRNO",` to be `"defaultAction": "SCMP_ACT_TRACE",`. A `sed` command is included below for your convenience.
+1. Edit the `/etc/docker/seccomp.json` file and modify the line `"defaultAction": "SCMP_ACT_ERRNO",` to be `"defaultAction": "SCMP_ACT_TRACE",`. A `sed` command is included below for your convenience. For Docker Engine 1.13.1 (RHEL only), as the `/etc/docker/seccomp.json` file already exists you can proceed with restarting the docker engine after modifying the aforementioned line by running `sudo systemctl restart docker`.
```

Comment:
What if we tried something like this? 

I'm thinking we can reduce the text a bit if we do this because now it's more obvious that the command is going to do what we just told them to do. And then, I think it makes sense to move the Docker note after the `sed` command since a) it doesn't apply to everyone and b) it's something folks should only do after they run that command to update the `seccomp.json` file. Thoughts?  We'd need to test the build to make sure everything renders properly (I know sometimes markdown is finicky with formatting).

```suggestion
1. In the `/etc/docker/seccomp.json` file, change `"defaultAction": "SCMP_ACT_ERRNO",` to `"defaultAction": "SCMP_ACT_TRACE",`. 
     
      ```
       sudo sed -i 's/"defaultAction":\s*"SCMP_ACT_ERRNO"/"defaultAction": 
      "SCMP_ACT_TRACE"/1' /etc/docker/seccomp.json
       ```
    **Docker Engine 1.13.1 (RHEL only):**  After modifying the `/etc/docker/seccomp.json` file, run `sudo systemctl restart docker` to restart the Docker engine.
```

Reply:
Love it! Thank you so much!